nncf/quantization/algorithms/smooth_quant/openvino_backend.py

```diff
@@ -170,3 +166,29 @@ def scale_insertion_command(
         source_node: NNCFNode, scale_value: np.ndarray, port_id: int, nodes: List[NNCFNode]
     ) -> OVMultiplyInsertionCommand:
         return OVCommandCreator.multiply_insertion_command(source_node, nodes, port_id, scale_value)
+
+    @staticmethod
+    def _get_activation_channel_axis(node: NNCFNode, port_id: int) -> int:
+        """
+        Returns axis number of the activation tensor which correspond to it channel.
+
+        :param node: NNCFNode instance.
+        :param port_id: Specified input port id.
+        :return: Channel axis number.
+        """
+        channel_axis = 1
+
+        if node.metatype == OVMatMulMetatype:
```

Comment:
Do you support matmul operation with a vector? I mean [N] x [1, K] = [N, K] or [N, M] x [M] = [N] or [N]x[K] = [N, K].

Reply:
Are there any examples of models that contain such case and can be tested?

Reply:
Test was added